website/templates/profile/notifications.mako

```diff
@@ -44,13 +45,23 @@
                     <!-- Flashed Messages -->
                     <div data-bind="html: message, attr: {class: messageClass}"></div>
             </div><!--view model scope ends -->
-            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><h3 class="panel-title">Configure Notification Preferences</h3></div>
+                <form id="selectNotifications" class="osf-treebeard-minimal">
+                    <div id="grid"></div>
+                    <div class="help-block" style="padding-left: 15px">
+                            <p id="configureNotificationsMessage"></p>
+                    </div>
+                </form>
```

Comment:
Actually it's not very hard to implement. Treebeard has a notify class that is used for this purpose, it shows a message replacing either the entire row or a column and disappears after 2 seconds (or what you set). I can show you how to implement it, but fangorn.js has many examples, like: 
```javascript
        item.notify.update(msgText, 'success', column, 0);
```
It also has good documentation here: 
https://github.com/caneruguz/treebeard/wiki/Notify